packages/snowpack/src/util.ts

```diff
@@ -177,22 +177,46 @@ export async function openInBrowser(
     : /brave/i.test(browser)
     ? appNames[process.platform]['brave']
     : browser;
-  if (process.platform === 'darwin' && /chrome|default/i.test(browser)) {
+  const isMac = process.platform === 'darwin' && /chrome|default/i.test(browser);
```

Comment:
NIT: This might be a bit confusing if used elsewhere since we're also asserting a browser?

Reply:
I actually didn’t realize we were asserting a browser. This is true on my machine and Chrome’s not my default. Maybe I’ll look into why that is.

Reply:
(this is my first experience with this code) OK so the `browser` here is the setting from the Snowpack config. Which is `default` by default (unsurprisingly). So basically this will fire on every MacOS machine _unless_ they’ve specified a non-Chrome browser to open.

Point taken though—I can change the var name